IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedBldgElements/Entities/IfcArchElement/DocEntity.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcArchElement" UniqueId="497395c2-219d-4d42-9eaf-1a8c3f3967cd" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="PredefinedType" UniqueId="140250ca-4d21-4dca-9f29-7b99e1c6857b" DefinedType="IfcArchElementTypeEnum" AttributeFlags="1">
+			<Documentation>Predefined generic type for an arch element that is specified in an enumeration. There may be a property set given specifically for the predefined types.
```

Comment:
Did we not have a more generic common description that we used? This, seems to me, is not it. 🤔

Reply:
There are a few...A generic one I found is: "Specifies the type for which the value is selected from a predefined type enumeration. This type may associate additional specific property sets".

Reply:
This wording "Predefined generic type" bothers me a bit. What is generic about the predefined type? 🤔

Reply:
> This wording "Predefined generic type" bothers me a bit. What is generic about the predefined type? 🤔

Please check now.